composer/profiler/profiler.py

```diff
@@ -108,6 +110,21 @@ def __init__(
         self.schedule = schedule
         self.state = None
         self._callbacks: List[Callback] = []
+        self.remote_filenames: List[str] = []
+        # First, add each remote file name to self.remote_filenames to create RemoteUplaoderDownloader logger in trainer. [s3://bucket/path/to/file]
+        # Then modify remote file name to be a local path to pass into torch_profiler and system_profiler. e.g: path/to/file
+        if torch_prof_remote_file_name:
+            self.remote_filenames.append(torch_prof_remote_file_name)
+            _, _, torch_prof_remote_file_name = parse_uri(torch_prof_remote_file_name)
```

Comment:
this seems unused?

Reply:
Used here: https://github.com/mosaicml/composer/pull/2653/files/b29a586e15a4ae60a77fe4b0f075abaf44811a24#diff-f264eab2433404bea45e45f53b2562cf2b3083199c4618873d973ca43adb1334R141